db/merge_helper.h

```diff
@@ -80,7 +80,8 @@ class MergeHelper {
   Status MergeUntil(InternalIterator* iter,
                     CompactionRangeDelAggregator* range_del_agg = nullptr,
                     const SequenceNumber stop_before = 0,
-                    const bool at_bottom = false);
+                    const bool at_bottom = false,
+                    const bool allow_data_in_errors = true);
```

Comment:
All other arguments have comments to explain it. Should do it for the new one too.